### PR TITLE
singlesweep CLI app, support uncompressed keys, RBF detection, and general cleanup

### PR DESCRIPTION
**Pull request by mflaxman**
I've tested this a bunch with Electrum (good UI for transaction selection, RBF, etc) and it plays nicely with the PSBTs it generates.

### CODE REVIEW

**Comment by mflaxman** [comment]
Screenshot:
Screen Shot 2021-04-30 at 9 34 36 PM

Values to use to try it out yourself:
WIF:
`cMahea7zqjxrtgAbB7LSGbcQUr1uX1ojucUiyLpRMKHBKBAc65Wz`
PSBT:
`cHNidP8BAFUCAAAAAa6Az7H1yBEqLWGUz/lVY+m2yRc97C62TBEUE2lUuqR6AQAAAAD9////AdAkAAAAAAAAGXapFAqVf1Opg7FBWzFEvqup/tfTgIn8iKxdHB4AAAEA4QIAAAAAAQEx7RGr1hBisiqsXl83/6qi/Dh5RsQH7LvQUeDk8ByHNwAAAAAA/v///wI1yu78CAAAABYAFDxuopFQSM9MOuAi8Z0PZn3wqVDbECcAAAAAAAAZdqkUCpV/U6mDsUFbMUS+q6n+19OAifyIrAJHMEQCIA4Hg21Y9pc6R4qJTKr69IApZT5wgBOqTSxGvXoeOLtiAiAOdDJnAQFVYUOxgZF7W8hJjGQplb65LEz/lfGDExvWvQEhA9XeHN/0faG7p8yEG8B+scrq6w/i8qgpILc72nPzZJbWXBweAAAA`

File: multiwallet.py [hunk 1]
```diff
@@ -356,6 +356,7 @@ class MyPrompt(Cmd):
     )
     intro = (
         "Welcome to multiwallet, the stateless multisig bitcoin wallet.\n"
+        "This tool is free and there is NO WAARRANTY OF ANY KIND.\n"
```
Comment (jimmysong) [inline multiwallet.py:359]: spelling mistake in WAARRANTY

**Review by jimmysong — APPROVED** [review]
I didn't look too closely at the logic, but it looks like you have something to sign some WIFs. Merge whenever you feel it's ready.

File: multiwallet.py (hunk 1 above)
Comment (mflaxman) [reply multiwallet.py]: Haha, I'm normally a decent speller but when I'm writing code I spell like a child 🤦‍♂️

Fixed.

**Comment by mflaxman** [comment]
Thanks @jimmysong. I'm a big fan of these CLI apps, so HT to you for originally suggesting that with multiwallet.

**Comment by mflaxman** [comment]
Oof, just realized that this merge button didn't nicely auto-squash it down. The danger of merging in code late at night!

**Pull request merged**